epick_description/urdf/epick_macro.xacro

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+
+  <xacro:macro name="robotiq_epick"
+               params="prefix
+                       parent
+                       *origin
+                       extension_radius:=0.0
+                       extension_length:=0.0
+                       suction_cup_radius:=0.012
+                       suction_cup_height:=0.015">
```

Comment:
For consistency with the extension, can this also be `suction_cup_length`?

Reply:
I would prefer keeping it this way. Technical specs for suction cups commonly use height and for extensions length.